crypto/evp_extra/p_kem.c

```diff
@@ -55,6 +93,77 @@ static int pkey_kem_keygen(EVP_PKEY_CTX *ctx, EVP_PKEY *pkey) {
   return 1;
 }
 
+static int pkey_kem_encapsulate_deterministic(EVP_PKEY_CTX *ctx,
+                                              uint8_t *ciphertext,
+                                              size_t  *ciphertext_len,
+                                              uint8_t *shared_secret,
+                                              size_t  *shared_secret_len,
+                                              const uint8_t *seed,
+                                              size_t *seed_len) {
+  KEM_PKEY_CTX *dctx = ctx->data;
+  const KEM *kem = dctx->kem;
```

Comment:
Same null checks before de-referencing public pointers.

Reply:
Added in [6aa7b7d](https://github.com/aws/aws-lc/pull/1578/commits/6aa7b7d2e5b3f035ad8fae576f529744a8dd1f04).

Reply:
Looks like this particular check might have been missed.

Reply:
got it this time [3578aee](https://github.com/aws/aws-lc/pull/1578/commits/3578aeef15c7532dc44a9604f8d9dd2e7793dae3)